id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoServiceImpl.java

```diff
@@ -745,9 +860,6 @@ private void issueCredential(String enryptedUin, String uinHash, String uinStatu
 			credStatus.setIdExpiryTimestamp(uinStatus.contentEquals(activeStatus) ? null : expiryTimestamp);
 			credStatus.setCreatedBy(IdRepoSecurityManager.getUser());
 			credStatus.setCrDTimes(DateUtils.getUTCCurrentDateTime());
-			if(enableConventionBasedId && (requestId != null)) {
```

Comment:
This should not be removed. This one is newly added, and this is not removed for resident.

Reply:
Added.